### PR TITLE
dont track health check

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn review please

File: cms/tests/test_events.py [hunk 1]
```diff
@@ -109,3 +109,45 @@ def test_ga_context_decorator(self, mock_task):
...
+            'HTTP_HOST': 'some.site.com',
+            'REMOTE_ADDR': '192.0.0.1',
+        })
+        self.assertEqual(mock_task.call_count, 1)
+
+        self.app.post('/search/', status=200, extra_environ={
+            'HTTP_HOST': 'some.site.com',
+            'REMOTE_ADDR': '192.0.0.1',
+        })
+        self.assertEqual(mock_task.call_count, 1)
```
Comment (smn) [inline cms/tests/test_events.py]: I was expecting this to be `2` since `/search/` isn't excluded?

Comment (miltontony) [inline cms/tests/test_events.py]: @smn I'm using a "comma separated list" `settings['ga.excluded_paths'] = '/locale/change/,/search/'`

Comment (smn) [inline cms/tests/test_events.py]: Ah 😀 👍

On Tuesday, 6 October 2015, Milton Madanda <notifications@github.com> wrote:

> In cms/tests/test_events.py
> <https://github.com/universalcore/unicore-cms/pull/110#discussion_r41228573>
> :
>
> > +        settings = self.get_settings(self.workspace)
> > +        settings['ga.profile_id'] = 'UA-some-id'
> > +        settings['ga.excluded_paths'] = '/locale/change/,/search/'
> > +        self.app = self.mk_app(self.workspace, settings=settings)
> > +
> > +        self.app.post('/locale/change/', status=200, extra_environ={
> > +            'HTTP_HOST': 'some.site.com',
> > +            'REMOTE_ADDR': '192.0.0.1',
> > +        })
> > +        self.assertEqual(mock_task.call_count, 1)
> > +
> > +        self.app.post('/search/', status=200, extra_environ={
> > +            'HTTP_HOST': 'some.site.com',
> > +            'REMOTE_ADDR': '192.0.0.1',
> > +        })
> > +        self.assertEqual(mock_task.call_count, 1)
>
> @smn <https://github.com/smn> I'm using a "comma separated list" settings['ga.excluded_paths']
> = '/locale/change/,/search/'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/universalcore/unicore-cms/pull/110/files#r41228573>.
>


-- 
Simon de Haan | Chief Engineer Praekelt Foundation

praekeltfoundation.org | praekelt.com
M: +27 (0) 76 449 3806  |  http://twitter.com/smn

**Pull request merged**